section2/exercises/functions.js

```diff
@@ -35,18 +41,35 @@ printName();
 // The function should print out the value of the String that was passed in.
 // Then, call the function 3 times, each time, passing it a different name.
 
+function greetMe(name, lastName) {
```

Comment:
🚨 If you are looking for an example of where you practiced functions/methods that "take an argument"